Sources/Rendering/Core/ImageCPRMapper/index.d.ts

```diff
@@ -62,6 +62,21 @@ export interface vtkImageCPRMapper extends vtkAbstractMapper3D {
 	 */
 	setUseUniformOrientation(useUniformOrientation: boolean): boolean;
 
+	/**
+	 * A point used to offset each line of pixel in the rendering
+	 * The line of pixel is offseted such as the center of the line is as close as possible to the center point
+	 * This can be used in combination with @see getUseUniformOrientation and a custom distance function for @see getOrientedCenterline to visualize a CPR in projected mode or stretched mode
+	 * Defaults to null.
+	 * @returns the center point
+	 */
+	getCenterPoint(): Nullable<vec3>;
```

Comment:
`Nullable<Vector3>` ?

Reply:
The difference between the two is:
`type Vector3 = [number, number, number];`
and
`type vec3 = [number, number, number] | Float32Array;`

The code I made accepts both Float32Array and Array and `vec3` are already used in a lot of typescript headers
Why do you think I should use `Vector3` instead of `vec3` ?

Reply:
Is keeping `Nullable<vec3>` fine @daker?

Reply:
Yes, sorry i didn't see your reply